src/pages/permissions/actions/DeleteIdpGroupsModal.tsx

```diff
@@ -25,9 +26,15 @@ const DeleteIdpGroupsModal: FC<Props> = ({ idpGroups, close }) => {
       ? deleteIdpGroup(idpGroups[0].name)
       : deleteIdpGroups(idpGroups.map((group) => group.name));
 
-    const successMessage = hasOneGroup
-      ? `IDP group ${idpGroups[0].name} deleted.`
-      : `${idpGroups.length} IDP groups deleted.`;
+    const successMessage = hasOneGroup ? (
+      <>
+        IDP group{" "}
+        <ResourceLabel bold type="auth-group" value={idpGroups[0].name} />{" "}
```

Comment:
I think this should be a different type than auth-group.

Reply:
Shall we introduce a new `idp-group` resource type?

Reply:
sounds good, done. Both `auth-group` and `idp-group` still map to the same icon though.